account_invoice_rounding_by_currency/models/company.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Alessio Gerace <alessio.gerace@agilebg.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields
+
+
+class RoundingByCurrency(models.Model):
+    _name = 'company.rounding'
+
+    company_id = fields.Many2one('res.company', 'Company', required=True)
+    currency_id = fields.Many2one('res.currency', 'Currency', required=True)
+    tax_calculation_rounding = fields.Float('Tax Rounding Precision')
+    tax_calculation_rounding_account_id = fields.Many2one(
+        'account.account',
+        'Tax Rounding Account',
+        domain=[('type', '<>', 'view')]
+    )
+    tax_calculation_rounding_method = fields.Selection(
+        [
+            ('swedish_round_globally', 'Swedish Round globally'),
+            ('swedish_add_invoice_line', 'Swedish Round by adding a line'),
+        ],
+        string='Tax Calculation Rounding Method',
+        help="If you select 'Round per line' : for each tax, the tax "
+        "amount will first be computed and rounded for each "
+        "PO/SO/invoice line and then these rounded amounts will be "
+        "summed, leading to the total amount for that tax. If you "
```

Comment:
It does but you have the other meaning as well. Check #3 here: http://www.thefreedictionary.com/sum+up
"sum up - determine the sum ofsum up - determine the sum of; "Add all the people in this town to those of the neighboring town""

Anyway, I am not here to be picky and I am not native English so it is up to you :)